### DISCUSSION TITLE
Carbamazepine PBPK model

### DISCUSSION THREAD

**Discussion by LMFuhr** [Open-Systems-Pharmacology/Forum#740] [Announcement]
Dear OSP-community,

We have been working on a parent-metabolite PBPK model of carbamazepine and its main metabolite carbamazepine-10,11-epoxide. 

The model was developed to be used in DDI modeling with carbamazepine as CYP3A4 and CYP2B6 substrate and inducer and was evaluated in a DDI network with 

- erythromycin (CYP3A4 inhibitor)
- alprazolam and simvastatin (CYP3A4 substrates)
- bupropion (CYP2B6 substrate)
- efavirenz (CYP3A4 and CYP2B6 substrate and inducer)

The corresponding manuscript, including comprehensive documentation, has been published online in Pharmaceutics and the carbamazepine PBPK model as well as PK-Sim projects of the predicted DDI studies can be found [here](https://github.com/Open-Systems-Pharmacology?q=carbamazepine&type=&language=&sort=).

Related publication:
[Fuhr, L.M.; Marok, F.Z.; Hanke, N.; Selzer, D.; Lehr, T. Pharmacokinetics of the CYP3A4 and CYP2B6 Inducer Carbamazepine and Its Drug–Drug Interaction Potential: A Physiologically Based Pharmacokinetic Modeling Approach. *Pharmaceutics* **2021**, *13*, 270.](https://www.mdpi.com/1999-4923/13/2/270)

Best regards,
Laura

**Comment by AndreDlm** [comment]
Thanks @LMFuhr, that's very helpful.
Do you have any plans to upload the evaluation plan soon?
